File: ASP_MVCNetExample/Angular/src/app/shared/MovieModel.ts
export default class MovieModel {
  id: number;
  name: string;
  releaseDate: string;

  constructor(id: number, name: string, releaseDate: string) {
    this.id = id;
    this.name = name;
    this.releaseDate = releaseDate;
  }
}

File: ASP_MVCNetCoreExample/Models/MovieModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ASP_MVCNetCoreExample.Models
{
    public class MovieModel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Director { get; set; }
        public string ImageUrl { get; set; }
        public bool Favourite { get; set; }
        public DateTime ReleaseDate { get; set; }
    }
}
